src/forms/Textarea.jsx

```diff
@@ -113,29 +113,30 @@ class Textarea extends React.Component {
 		};
 
 		return (
-			<div className="inputContainer">
-				{label &&
-					<label className={classNames.label} htmlFor={id}>
-						{label}
-					</label>
-				}
-
-				<textarea
-					type='text'
-					name={name}
-					required={required}
-					className={classNames.textarea}
-					onChange={this.onChange}
-					rows={rows}
-					ref={(textarea) => {this.textarea = textarea;}}
-					style={{ ...style, ...heightConstraints }}
-					id={id}
-					value={this.state.value}
-					{...other}
-				/>
-
-				{ maxLength && <p className='text--tiny text--secondary align--right charCount'>{parseInt(maxLength - this.state.value.length)}</p> }
-
+			<div>
+				<div className="inputContainer">
```

Comment:
is `inputContainer` class providing anything? should we remove? should we move it one level up ?

Reply:
It's there on purpose - it does `position: relative;`, keeping the character count at the bottom right of the input